extensions/adobe/experience/adcloud/searchadvertisingadgroup.schema.json

```diff
@@ -0,0 +1,121 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/searchadvertisingadgroup",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising AdGroup",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Search Advertising Ad Group is a logical collection or grouping of ads.",
+  "definitions": {
+    "searchadvertisingadgroup": {
+      "properties": {
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
```

Comment:
Why cant you use same _id that comes out of record here ?

Reply:
Most of the entities (search) have two identifiers - one is ad-cloud identifier which uniquely identifies the record across search engines and other is external identifier (unique per search engine).  _id that comes out of record is used as an internal identifier.

Reply:
What value would you populate in the id that comes from the record?

Reply:
@cmathis Internal Identifier comes from our product (AMO) databases. It is an unique Id across all search engine data that we work with.